server/src/test/java/org/opensearch/index/mapper/NumberFieldMapperTests.java

```diff
@@ -215,23 +218,6 @@ public void testIgnoreMalformed() throws Exception {
         }
     }
 
-    /**
```

Comment:
The test was added by elastic/elasticsearch#41261 and the only reason it was passing after #4494 is because default index-level setting for `ignore_malformed` was `false`